corebehrt/azure/__main__.py

```diff
@@ -0,0 +1,91 @@
+"""
+Command-line utility for starting CoreBEHRT jobs on Azure clusters.
+Requires installation of azure-ml-ai python package and a valid Azure workspace. 
+"""
+
+import sys
+import argparse
+
+from corebehrt.common.config import load_config
+
+from corebehrt.azure.components import create_data, create_outcomes, pretrain, finetune
+
+from . import util
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        prog="corebehrt.azure", description="Run corebehrt jobs and pipelines in Azure"
+    )
+
+    # Global options
+    parser.add_argument(
+        "-e",
+        "--experiment",
+        type=str,
+        default="corebehrt_runs",
+        help="Experiment to run the job in.",
+    )
+    parser.add_argument(
+        "-c",
+        "--compute",
+        type=str,
+        help="Compute target to use. Default depends on job/component.",
+    )
+    parser.add_argument(
+        "-o",
+        "--register_output",
+        type=str,
+        action="append",
+        default=[],
+        help="If an output should be registered, provide a name for the Azure asset using the format '--register_output <input>=<name>'.",
+    )
+
+    # Sub-parsers
+    subparsers = parser.add_subparsers(dest="call_type")
+    # Job parser
+    job_parser = subparsers.add_parser("job", help="Run a single job.")
+    job_parser.add_argument(
+        "JOB",
+        type=str,
+        choices=("create_data", "create_outcomes", "pretrain", "finetune_cv"),
+        help="Job to run.",
+    )
+    job_parser.add_argument(
+        "-c",
```

Comment:
Can we change the flage to something else to not introduce conflicts with the -c compute flag?

Reply:
I thought about this, it should not cause a conflict as it is for the sub command - but I think the change is good for clarity anyways. I will change it

Reply:
I thought about this, it should not cause a conflict as it is for the sub command - but I think the change is good for clarity anyways. I will change it

Reply:
I changed compute to be non-optional instead, so no conflicts.

Reply:
I changed compute to be non-optional instead, so no conflicts.

Reply:
ok